python/mxnet/gluon/block.py

```diff
@@ -1271,8 +1275,13 @@ def export(self, path, epoch=0, remove_amp_cast=True):
             if name in arg_names:
                 arg_dict['arg:%s'%name] = param._reduce()
             else:
-                assert name in aux_names
-                arg_dict['aux:%s'%name] = param._reduce()
+                if name.endswith('running_mean') or name.endswith('running_var') \
+                    or name.endswith('moving_mean') or name.endswith('moving_var'):
+                    assert name in aux_names
+                    arg_dict['aux:%s'%name] = param._reduce()
```

Comment:
Where do we ensure that no aux parameter with a name different to `running_mean`, `running_var`, etc is missing from the resulting params file?